Android: Modules build with 11.1.0.GA not working in apps

### I have searched and made sure there are no existing issues for the issue I am filing

- [X] I have searched the existing issues

### Description

Gradle tools in master were updated to 7.2.2. Building a module with it breaks apps:

```
[ERROR] E/ga.testfirebase: No implementation found for void com.williamrijksen.onesignal.TiModuleBootstrap.nativeBootstrap() (tried Java_com_williamrijksen_onesignal_TiModuleBootstrap_nativeBootstrap and Java_com_williamrijksen_onesignal_TiModuleBootstrap_nativeBootstrap__)
```

Lowering it back to 7.0.4 again makes it work again.


**Fix:**
11.1.X: https://github.com/tidev/titanium_mobile/pull/13572
Master: https://github.com/tidev/titanium_mobile/pull/13571

--- a/android/templates/build/root.build.gradle
+++ b/android/templates/build/root.build.gradle
@@ -7,7 +7,7 @@ buildscript {
 		mavenCentral()
 	}
 	dependencies {
-		classpath 'com.android.tools.build:gradle:7.2.0'
+		classpath 'com.android.tools.build:gradle:7.0.4'
 		classpath 'com.google.gms:google-services:4.3.10'
 		classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
 	}

